src/openrct2/rct12/Limits.h

```diff
@@ -33,7 +33,7 @@ namespace RCT12::Limits
     constexpr const uint8_t FinanceGraphSize = 128;
 
     constexpr const uint16_t MaxUserStrings = 1024;
-    constexpr const uint8_t MaxUserStringLength = 32;
+    constexpr const uint8_t MaxUserStringLength = 64;
```

Comment:
This constant is for RCT2 strings, by the 'new' OpenRCT2 save format.

Reply:
Do you mean that I can't _just_ change ride name lengths, and that everything else would have to change too? I wholly appreciate that, however my sole intention is making it so that longer ride name lengths can be inputted to make full use of coloured text - which has been part of RCT2 since the beginning, but has only recently become more accessible since Deuerklink's video on them.

Reply:
This specific constant is for the import of RCT1/RCT2 save files, changing this will break loading sv4/sv6 files. You need to dig a bit deeper, this limit is probably in the UI code somewhere, as far as I can tell there is no restriction for how long a ride name can be.